examples/ex11-jacobi.cpp

```diff
@@ -338,9 +342,9 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
     //
     // Jacobi Iteration 
     //
-    RAJA::nested::forall(jacobiCUDANestedPolicy{},
+    RAJA::nested::forall<jacobiCUDANestedPolicy>(
                          RAJA::make_tuple(jacobiRange,jacobiRange),
-                         [=] __host__ __device__  (RAJA::Index_type m, RAJA::Index_type n) {
+                         [=] __device__  (RAJA::Index_type m, RAJA::Index_type n) {
```

Comment:
Yes, we should be using the macro in the examples. Another check box to fill in as we rework things.

Reply:
agreed. I will change those when I do a cleanup pass of this branch

Reply:
When you're going through the examples, will you do a sanity check on the code comments describing the different kernels. They may not be entirely correct with the new nested policies.  For example, the descriptions of NESTED_EXEC_POL4 and NESTED_EXEC_POL5 in ex4-matrix-multiply.cpp need to be updated I think. I wrote these originally, so it would be code to have another pair of eyes look at them.  When the examples are all converted and checked out, let me know. I'll go in and rework the tutorial discussion and fix the statements that pull in the code snippets.  Thanks.